<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Helicopter_PanelRadio</class>
 <widget class="QWidget" name="Helicopter_PanelRadio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helicopter_PanelRadio</string>
  </property>
  <widget class="QGroupBox" name="APK15">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>1</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>АРК-15</string>
   </property>
   <widget class="QPushButton" name="t_rk_ramButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>РАМКА</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="t_rk_ctrlButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>УПР</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>179</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QCheckBox" name="freqHlebCB">
       <property name="text">
        <string>Частота Хлебодаровки </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="freqAktobeCB">
       <property name="text">
        <string>Частота Актобе</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>65</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>ТЛФ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="tlg_tlfSlider">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>ТЛГ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>181</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>ВЫКЛ  -  КОМ  -  АНТ  -  РАМ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="kom_ant_ramSlider">
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>101</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="text">
        <string>   1 -- КАНАЛ -- 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="chanel12Slider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>524</x>
     <y>1</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>АРК-УД</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>355</width>
      <height>189</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLineEdit" name="lineEdit_9">
           <property name="text">
            <string>Чувств</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="t_rkud_sensSlider">
           <property name="maximum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_10">
           <property name="text">
            <string>      М      </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLineEdit" name="lineEdit_11">
           <property name="text">
            <string>УКВ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="t_rkud_ukvSlider">
           <property name="maximum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_12">
           <property name="text">
            <string>ДЦВ</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="UP_lamp">
             <property name="text">
              <string>  УП</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHP_lamp">
             <property name="text">
              <string>  ШП</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="I_lamp">
             <property name="text">
              <string>   И</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_13">
           <property name="text">
            <string> ВЫКЛ-----УП-----ШП-----И-----РПК</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="t_rkudSlider">
           <property name="maximum">
            <number>4</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_14">
           <property name="text">
            <string> 1------2------3------4------5-------6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="t_chanelSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>6</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="t_contrButton">
         <property name="text">
          <string>КОНТР</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="t_lButton">
         <property name="text">
          <string>Л</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="t_rButton">
         <property name="text">
          <string>П</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>ГМК-1Гэ</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLineEdit" name="lineEdit_16">
         <property name="text">
          <string>СЕВ -- ЮГ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sev_ugSlider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_17">
         <property name="text">
          <string>0 -----300</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="contr0_300Slider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="eservLamp">
           <property name="baseSize">
            <size>
             <width>10</width>
             <height>5</height>
            </size>
           </property>
           <property name="text">
            <string> ЗАП</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mainLamp">
           <property name="text">
            <string>        ОСН</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="reserv_mainSlider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="longitude">
         <property name="minimum">
          <number>30</number>
         </property>
         <property name="maximum">
          <number>90</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="longLCD"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLineEdit" name="MkLINE">
         <property name="text">
          <string>  МК--- ГПК</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="mk_gpkSlider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_20">
         <property name="text">
          <string> Л - ЗК - П</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="zkSlider">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>501</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Тумблера с других панелей для тестов</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>179</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="kurSmalLCD"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="text">
        <string>КУР мал.стрелка</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="kurBigLCD"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="text">
        <string>КУР бол.стрелка</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="text">
        <string>УКР-СР-КР-ДР-РК1-РК2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="galSPU">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>6</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>101</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="text">
        <string>СПУ  --  РАДИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="SPU_RADIO">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>65</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_16">
     <item>
      <widget class="QSlider" name="t_gmkSlider">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_18">
       <property name="text">
        <string>ГМК</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>410</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>УВ-26</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-----БОРТ-----&gt;                        НАЛ-------ПРОГР</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QPushButton" name="uv26SBROS">
       <property name="text">
        <string>СБРОС ПРОГ</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSlider" name="uv26Prog">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="uv26Stop">
       <property name="text">
        <string>СТОП</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="uv26LCD"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="uv26L68">
       <property name="text">
        <string>L6.8</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSlider" name="uv26Bort">
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="uv26Pusk">
       <property name="text">
        <string>ПУСК</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="uv26L45">
       <property name="text">
        <string>L4.5</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="uv26L8">
       <property name="text">
        <string>L8</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
